app/src/main/java/com/github/sdpteam15/polyevents/util/ItemEntityAdapter.kt

```diff
@@ -2,24 +2,21 @@ package com.github.sdpteam15.polyevents.util
 
 import com.github.sdpteam15.polyevents.database.DatabaseConstant.ItemConstants.*
 import com.github.sdpteam15.polyevents.model.Item
-import com.github.sdpteam15.polyevents.model.ItemType
 
 object ItemEntityAdapter {
     fun toItemEntity(documentData: MutableMap<String, Any?>, id: String): Pair<Item, Int> =
         Pair(
             Item(
                 id,
                 documentData[ITEM_NAME.value] as String,
-                ItemType.valueOf(
-                    documentData[ITEM_TYPE.value] as String
-                )
-            ), (documentData[ITEM_COUNT.value] as Long).toInt()
+                documentData[ITEM_TYPE.value] as String),
+                (documentData[ITEM_COUNT.value] as Long).toInt()
```

Comment:
You can just cast as Int at first no ? Instead of casting as Long and then as Int ?

Reply:
![image](https://user-images.githubusercontent.com/25907972/115676967-5a24bb80-a350-11eb-9834-21ae1385da3a.png)
Casting directy as an Int leads to and error, so we must cast first as Long and then convert to Int